Makefile

```diff
@@ -0,0 +1,3 @@
+build:
+	docker build -t kubecrawler .
```

Comment:
This looks like a good first iteration. We could set this to a variable as well, so we would only need to change the name in one place if ever.
`IMAGE_NAME=kubecrawler
...
    docker build -t $(IMAGE_NAME) .
`

Reply:
Also, we want to make sure the image always has a unique tag using a version, build number, or timestamp.
`docker build -t $(IMAGE_NAME):$(VERSION)`